tests/message/test_tasks.py

```diff
@@ -109,6 +109,10 @@ def test_extra_data(self):
         self.assertTrue(extra_data['frames'])
         self.assertTrue(extra_data['output_format'])
 
+    def test_has_resources(self):
+        ctd = message.ComputeTaskDef()
```

Comment:
```suggestion
        ctd = message.tasks.ComputeTaskDef()
```

Reply:
IMHO This method only tests whether `FrozenDict` works as expected. I think it's not necessary.